go/vt/vtctl/workflow/switcher_dry_run.go

```diff
@@ -63,6 +63,16 @@ func (dr *switcherDryRun) deleteKeyspaceRoutingRules(ctx context.Context) error
 	return nil
 }
 
+func (dr *switcherDryRun) switchKeyspaceReads(ctx context.Context, types []topodatapb.TabletType) error {
+	var tabletTypes []string
+	for _, servedType := range types {
+		tabletTypes = append(tabletTypes, servedType.String())
+	}
+	dr.drLog.Logf("Switch reads for keyspace %s to keyspace %s for tablet types [%s]",
```

Comment:
IMO it's slightly nicer to `from keyspace to keyspace` here (from rather than for).